truss/trt_llm.py

```diff
@@ -0,0 +1,61 @@
+import logging
```

Comment:
would be cool if its in `config/trt_llm.py` so the top level is less crowded, but this is cool too